smack-extensions/src/main/java/org/jivesoftware/smackx/geoloc/GeoLocationManager.java

```diff
@@ -83,4 +88,53 @@ public static boolean isGeoLocationMessage(Message message) {
         return GeoLocation.from(message) != null;
     }
 
+    /**
+     * Returns the geolocation node in PubSub.
+     * 
+     * @return the geolocation PubSub node
+     * @throws InterruptedException
+     * @throws NotConnectedException
+     * @throws XMPPErrorException
+     * @throws NoResponseException
+     */
+    public LeafNode getNode()
```

Comment:
I'd probably move the method below the `sendGelocation()` and `stopPublishingGeolocation()` methods, since it's less important.